internal/peer/snapshot/submitrequest.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package snapshot
+
+import (
+	"context"
+	"fmt"
+
+	pb "github.com/hyperledger/fabric-protos-go/peer"
+	"github.com/hyperledger/fabric/bccsp"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// submitRequestCmd returns the cobra command for snapshot submitrequest command
+func submitRequestCmd(clientConnection *ClientConnection, cryptoProvider bccsp.BCCSP) *cobra.Command {
+	snapshotSubmitRequestCmd := &cobra.Command{
+		Use:   "submitrequest",
+		Short: "Submit a snapshot request for the given block number.",
```

Comment:
Not sure if this conveys the message... especially, `snapshot for the` part. How about "Submit a request for a snapshot at the specified block"?